api/api/utils/license.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from openverse_attribution.license import License
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_license_slug_enum(slug: str) -> tuple[str, License] | None:
+    """
+    Get the corrected license slug, and ``License`` enum for the given,
+    potentially invalid, license slug.
+
+    :param slug: the potentially invalid license slug
+    :return: the valid license slug
+    """
+
+    known_incorrect_licenses = {"publicdomain": "pdm", "sa": "by-sa"}
```

Comment:
This could go in `api/licenses/constants.py`.

Reply:
After 179182f, this is no longer needed as there is only the case of "publicdomain" to account for.